CONTRIBUTING.md

```diff
@@ -1061,26 +1061,26 @@ for the simple commands we use for this.
 
 To run clang-tidy locally, follow these steps:
 
-1. Install clang-tidy.
-We provide custom built binaries which have additional checks enabled. You can install it by running:
-```bash
-python3 -m tools.linter.clang_tidy.generate_build_files
-```
 We currently only support Linux and MacOS (x86).
 
-2. Install clang-tidy driver script dependencies
+1. Install lintrunner:
 ```bash
-pip3 install -r tools/linter/clang_tidy/requirements.txt
+pip3 install lintrunner
 ```
-
-3. Run clang-tidy
+2. Run lintrunner
 ```bash
-# Run clang-tidy on the entire codebase
-make clang-tidy
-# Run clang-tidy only on your changes
-make clang-tidy CHANGED_ONLY=--changed-only
+# Run lintrunner on the entire codebase
+lintrunner --take clang-tidy --all-files
+# Run lintrunner on your changes only
+lintrunner --take clang-tidy --changed-only
 ```
-This internally invokes our driver script and closely mimics how clang-tidy is run on CI.
+
+This will run a variety of linters on the entire codebase or on your changes only, respectively.
+One of the linters that lintrunner runs is clang-tidy.
+
+The reason why LLVM 8 is needed is because it is the version of LLVM that clang-tidy was tested with.
```

Comment:
Sorry, can you please elaborate what you mean by that? As one can see from tools/linter/adapters/s3_init_config.json as of time of this writing, clang-tidy from llvm-15 is used